### ISSUE TITLE
Request a digital signature for nvda dmp.exe

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#16428]
### Is your feature request related to a problem? Please describe.
Using NVDA for my daily work within the enterprise, I need to request an exception for applications that are not digitally signed.
### Describe the solution you'd like
Request that nvda dmp.exe be digitally signed, just like you do with other NVDA executables.
### Describe alternatives you've considered
none
### Additional context
I don't know that nvda dmp.exe is missing a signature, if there are other reasons please let me know, thanks.

**Comment by seanbudd** [comment]
NV Access doesn't currently own this code. Doing this requires taking ownership and forking that project. This is something we intend to do.

**Comment by cary-rowen** [comment]
I noticed that @codeofdusk provided #16508, what does @codeofdusk think about granting NV Access the rights to own this code? What challenges will exist?

Thanks

**Comment by seanbudd** [comment]
Closing as fixed by #18480
